src/main/java/org/example/homework11/Task1.java

```diff
@@ -0,0 +1,20 @@
+package org.example.homework11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Task1 {
+    public static void main(String[] args) {
+        String string = "hjfhfkjfjuk";
+        System.out.println(countLetters(string));
+    }
+
+    private static Map<Character, Integer> countLetters(String input) {
+        Map<Character, Integer> result = new HashMap<>();
+        for (Character letter : input.toCharArray()) {
+            result.putIfAbsent(letter, 0);
```

Comment:
```suggestion
        result.put(letter, result.getOrDefault(letter, 0) + 1);
```
it does the same but in one line